A tutorial on how to create a 3Box chatbox plugin.md

```diff
@@ -0,0 +1,11 @@
+# Content
+A tutorial on how to create a 3Box chatbox plugin
+
+Outline 
+
+1). An overview of 3box chatbox plugin
+2). Introduction to 3Box chatbox plugin
```

Comment:
@writeprovidence What's the difference between points 1, 2 and 3?

Maybe start with a general, but brief introduction to 3Box plugins first as point 1?

Reply:
Yes @ChrisChinchilla , that is the reason why i made it that way. i don't know if its fine by you.

Reply:
OK, that doesn't really answer my question, could you update the outline to clarify please?